src/resource/kv-store.c

```diff
@@ -363,7 +363,7 @@ static int _hash_update_fn(const char *               key,
 void *kv_store_set_value(sid_resource_t *          kv_store_res,
                          const char *              key,
                          void *                    value,
-                         size_t                    value_size,
+                         size_t                    value_count,
```

Comment:
@prajnoha maybe  there a context where this is a size and not a count?

Reply:
This is meant to be "vector item count" in case we have a vector (value flagged as `KV_STORE_VALUE_VECTOR`) - then there's size inside the `struct iovec` and its`iov_len` field for each vector item.

It's "real size" in case it's the usual scalar value (so not flagged as `KV_STORE_VALUE_VECTOR`). I used "size" for the name of the parameter which seemed to be more generic and could incorporate both vector and non-vector case...

Reply:
I see.  I'll drop the commit.